### PR TITLE
Add Documentation for Linux Environment Setup

### PR DESCRIPTION
**Pull request by paulhazen**
This PR adds two documents instructing contributors on how to set up a Linux environment (optionally on Hyper-V) in which to make contributions to the EOS Plugin for Unity.

Corresponding sections were added to `README.md` linking to the two guides that were added.

### CODE REVIEW

File: docs/dev_env/Ubuntu_Development_Environment.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+This is the second part of the guide for using Hyper-V to set up a Linux environment on your Windows PC that can be utilized for testing and developing the EOS Plugin. This guide _can_ however be used on its own as a guide to setting up a new Ubuntu 18.04 operating system for making contributions to the EOS Plugin.
+
+## Setup your Linux Guest VM in Hyper-V
+
+If you want to set up your linux environment inside a virtual machine, and have not yet done so, follow [this](/docs/dev_env/HyperV_Linux_Guest_VM.md) guide first.
+
+## How to expand Hyper-V disk space for Ubuntu (if using Hyper-V)
+
+This is for after you have Ubuntu installed. It is recommended to set your disk space to an appropriate size before you install the OS.
```
Comment (Jason-J-PEW) [inline docs/dev_env/Ubuntu_Development_Environment.md]: You should add instructions to the Hyper-V setup process to adjust drive size, so people don't end up needing this.

File: docs/dev_env/Ubuntu_Development_Environment.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+1. Go to File ➝ Build Settings
+2. Switch the platform to "Dedicated Server."
+3. Click on the button labeled "Player Settings..." at the bottom left.
+4. Go to Tools ➝ EOS Automated Test Settings
+5. Select the previously created `Builds/Server` directory for the "Test Server Directory" field.
+
+## Optional:
+
+Set up a shared folder between Linux and Windows. Follow these instructions: [https://linuxhint.com/shared\_folders\_hypver-v\_ubuntu\_guest/](https://linuxhint.com/shared_folders_hypver-v_ubuntu_guest/)
```
Comment (Jason-J-PEW) [inline docs/dev_env/Ubuntu_Development_Environment.md]: Should these be in the Hyper-V section? It seems like you're setting up a clear delineation between Hyper-V setup and regular linux setup with the 2 different files, but there is still a lot of Hyper-V related stuff in the regular linux file. Should these be more separated?

**Review by Jason-J-PEW — CHANGES_REQUESTED** [review]

File: docs/dev_env/Ubuntu_Development_Environment.md (hunk 1 above)
Comment (paulhazen) [reply docs/dev_env/Ubuntu_Development_Environment.md]: Good call - just added those, and moved this section to the end as a safety.

File: docs/dev_env/Ubuntu_Development_Environment.md (hunk 2 above)
Comment (paulhazen) [reply docs/dev_env/Ubuntu_Development_Environment.md]: Good catch. I moved these to the other doc.

**Review by Jason-J-PEW — APPROVED** [review]

**Review by Raxnam — APPROVED** [review]

**Pull request merged**